activities/safety_reaction.md

```diff
@@ -0,0 +1,80 @@
+## Disable automatic back-up when Kuri is on the dock
+
+This activity modifies kuri_edu/src/kuri_edu/safety_controller.py to disable
+the back-up reaction when Kuri is on the dock
+
+### Background
+
+Stopping the robot from moving after a bump event is handled by low level
+firmware.  When a bump is detected, the firmware stops the wheels until
+the safety controller node overrides the lock-out, backs the robot up,
+re-enables safety lock-outs, and clears the safety condition.
+
+```
+    # Override the safety status and back the robot up
+    self._safety_client.safety_override(current_status)
+    self._back_up(rate)
+    self._safety_client.safety_override(self.UNHANDLED_EVENTS)
+    self._safety_client.safety_clear(current_status)
+
+```
+
+You can verify that the safety controller is working by tapping the bump shell
+when the kuri_edu software is running.  Kuri will back up about an inch
+automatically
+
+### Activity
+
+Modify the safety controller to supress the back-up when kuri is on the dock.
+
+Some tips:
+  - Create a new file kuri_edu/src/kuri_edu/dock_monitor.py that contains a
+DockMonitor class.  This class can subscribe to the /mobile_base/power topic 
+and keep track of the latest value of `dock_present`
+  - Add the DockMOnitor class tokuri_edu/src/kuri_edu/\_\_init\_\_.py so it
```

Comment:
* There's a missing space here (tokuri_edu)
* Instead of escaping the `_`, you can format the file path as code: `kuri_edu/src/kuri_edu/__init__.py`